doc/source/api-reference/qibo.rst

```diff
@@ -1295,22 +1325,57 @@ Pauli-Liouville to Choi
 .. autofunction:: qibo.quantum_info.pauli_to_choi
 
 
-Choi to Pauli-Liouville
-"""""""""""""""""""""""
-
-.. autofunction:: qibo.quantum_info.choi_to_pauli
-
 
 Pauli-Liouville to Kraus
 """"""""""""""""""""""""
 
 .. autofunction:: qibo.quantum_info.pauli_to_kraus
 
 
-Kraus to Pauli-Liouville
-""""""""""""""""""""""""
+Pauli-Liouville to Chi-matrix
+"""""""""""""""""""""""""""""
 
-.. autofunction:: qibo.quantum_info.kraus_to_pauli
+.. autofunction:: qibo.quantum_info.pauli_to_chi
+
+
+Chi-matrix to Choi
+""""""""""""""""""
+
+.. autofunction:: qibo.quantum_info.chi_to_choi
+
+
+Chi-matrix to Liouville
+"""""""""""""""""""""""
+
+.. autofunction:: qibo.quantum_info.chi_to_liouville
+
+
+Chi-matrix to Pauli-Liouville
+"""""""""""""""""""""""""""""
+
+.. autofunction:: qibo.quantum_info.chi_to_pauli
+
+
+Chi-matrix to Kraus
+"""""""""""""""""""
+
+.. autofunction:: qibo.quantum_info.chi_to_kraus
+
+.. note::
+    Due to the spectral decomposition subroutine in this function, the resulting Kraus
+    operators :math:`\{K_{\alpha}\}_{\alpha}` might contain global phases. That
+    implies these operators are not exactly equal to the "true" Kraus operators
+    :math:`\{K_{\alpha}^{(\text{ideal})}\}_{\alpha}`. However, since these are
+    global phases, the operators' actions are the same, i.e.
+
+    .. math::
+        K_{\alpha} \, \rho \, K_{\alpha}^{\dagger} = K_{\alpha}^{\text{(ideal)}} \, \rho \,\, (K_{\alpha}^{\text{(ideal)}})^{\dagger} \,\,\,\,\, , \,\, \forall \, \alpha
+
+.. note::
+    User can set ``validate_CP=False`` in order to speed up execution by not checking if
+    the Choi representation obtained from the input ``chi_matrix`` is completely positive (CP) and Hermitian.
+    However, that may lead to erroneous outputs if ``choi_super_op`` is not guaranteed to be CP. We advise users
+    to either set this flag carefully or leave it in its default setting (``validate_CP=True``).
```

Comment:
I think now `validate_cp` is all lowercase letters. Could you change that here and the other please it appears? Thanks.